### PR TITLE
Adding China PWSH option

### PR DESCRIPTION
**Pull request by JefferyMitchell**

### CODE REVIEW

**Comment by PRMerger13** [comment]
@JefferyMitchell : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
#label:"aq-pr-triaged"

**Comment by dominicallen** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@dominicallen: I'm sorry - only the author of this article, @BrianBlanchard, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by BrianBlanchard** [comment]
#sign-off

File: docs/ready/landing-zone/implementation-options.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ The following table describes some of the implementation options for landing zon
...
 | [CAF enterprise-scale landing zone (Azure Gov)](../enterprise-scale/index.md) | Reference implementation that can be deployed to Azure Goverment Cloud. | Enterprise-scale | [Design principles](../enterprise-scale/design-principles.md) | [Deploy](https://portal.azure.us/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fmain%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fmain%2FeslzArm%2Feslz-portal.json) |
+| [CAF enterprise-scale landing zone (Azure China Regions)](../enterprise-scale/index.md) | Reference implementation that can be deployed to Azure Clouds in China. | Enterprise-scale | [Design principles](../enterprise-scale/design-principles.md) | [Deploy](https://github.com/Azure/Enterprise-Scale/blob/cf46ee50e2720a42bebdd5b43abc08738f349794/eslzArm/README-AzureChina.md) |
```
Comment (ShannonLeavitt) [inline docs/ready/landing-zone/implementation-options.md:40]:
```suggestion
| [CAF enterprise-scale landing zone (Azure China 21Vianet regions)](../enterprise-scale/index.md) | Reference implementation that can be deployed to Azure clouds in China. | Enterprise-scale | [Design principles](../enterprise-scale/design-principles.md) | [Deploy](https://github.com/Azure/Enterprise-Scale/blob/cf46ee50e2720a42bebdd5b43abc08738f349794/eslzArm/README-AzureChina.md) |
```
Fixing terminology per MS Cloud + AI style guide

**Comment by JanetCThomas** [comment]
@ShannonLeavitt @Court72 Any idea when this might go live?

**Comment by JefferyMitchell** [comment]
@ShannonLeavitt and @Court72, checking in on this one.

**Comment by BrianBlanchard** [comment]
#sign-off

**Comment by JefferyMitchell** [comment]
@ShannonLeavitt @Court72   Any update?

@BrianBlanchard @JanetCThomas

**Comment by Court72** [comment]
@JanetCThomas 

It looks like the public repo hasn't been synced with the private repo yet. The changes show in the public repo, but not in the live article.

Try reaching out to PubDesk. Maybe this repo isn't on their list?

See [How do the public repo changes get into the private repo?](https://review.docs.microsoft.com/en-us/help/contribute/contribute-vendor-public-prs?branch=main#how-do-the-public-repo-changes-get-into-the-private-repo)

**Comment by JefferyMitchell** [comment]
@JanetCThomas @JefferyMitchell @BrianBlanchard @dominicallen 
> 
> It looks like the public repo hasn't been synced with the private repo yet. The changes show in the public repo, but not in the live article.
> 
> Try reaching out to PubDesk. Maybe this repo isn't on their list?
> 
> See [How do the public repo changes get into the private repo?](https://review.docs.microsoft.com/en-us/help/contribute/contribute-vendor-public-prs?branch=main#how-do-the-public-repo-changes-get-into-the-private-repo)

Thats okay we can just delete this or cancel it however is best and I will submit it under cloud-adoption-framework-pr

**Pull request merged**